zebra-state/src/service/non_finalized_state/chain.rs

```diff
@@ -737,6 +737,16 @@ impl Chain {
             .collect()
     }
 
+    /// Returns the Sapling note commitment subtree of the tip of this [`Chain`],
```

Comment:
This documentation is ambiguous, because it doesn't say "latest subtree" or "completed subtree.

Orchard also needs the same changes.

Here's an edited copy of the docs for `Chain::sapling_subtree()`:
```suggestion
    /// Returns the Sapling [`NoteCommitmentSubtree`] if it was completed at the tip height.
```